crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -1409,6 +1409,15 @@ impl EntityCommands<'_> {
         self.queue(despawn())
     }
 
+    /// Despawns the entity.
+    /// This will not emit a warning if the entity does not exist.
+    ///
+    /// See [`World::try_despawn`] for more details.
```

Comment:
I think we're better off referencing [`Commands::despawn`] here and explaining the difference